source/educators/references/course_development/exercise_tools/multi_select.rst

```diff
@@ -32,14 +32,12 @@ As a best practice, be sure that all of the answer choices are unambiguous, and
 avoid trick questions. Multi-select problems with ambiguity can be frustrating to
 learners, especially if the problems have a limited number of attempts.
 
-=============================
+
 Example Multi-select Problem
-=============================
+****************************
 
-In the LMS, learners complete a multi-select problem by selecting the answer
-options that they believe are correct as well as leaving unselected the answer
-options that they believe are incorrect. An example of a completed multi-select
-problem follows.
+In the LMS, learners complete a multi-select problem by selecting the correct answers and leaving the incorrect ones unselected. An example of a completed multi-select
+problem follows.selecting the correct answers and leaving the incorrect ones unselected
```

Comment:
"follows.selecting" looks like a slight typo.

Reply:
Thanks for noticing. Adjustment made